docs-v2/reference/sdks/node.mdx

```diff
@@ -690,7 +690,7 @@ const records = await nango.listRecords<ModelName>({
 ```
 
 <Warning>
-`nango.getRecords()` is deprecated and  will be removed in future releases as it does not support efficient pagination. Please use `nango.listRecords()` detailed below.
+`nango.getRecords()` is deprecated and will be removed in future releases as it does not support efficient pagination. Please use `nango.listRecords()` detailed below.
```

Comment:
Note we could add `// @deprecated` to the node-client to make dev aware of this

Reply:
Done!